### PR TITLE
Add back lost optional unwrapping feature used in EagerExecution.swift.

### PR DESCRIPTION
**Pull request by pschuh**
This is needed to make the tensorflow-swift-bindings compile again.

### CODE REVIEW

**Review by dan-zheng — APPROVED** [review]

File: stdlib/public/TensorFlow/Utilities.swift [hunk 1]
```diff
@@ -176,3 +176,17 @@ public func _addScalarTensorsWithShape<Scalar: TensorFlowNumeric>(
 ) -> Tensor<Scalar> {
   return Raw.add(x, y)
 }
+
+// TODO: Consider revising the call sites where this is necessary to only need
+// UnsafeMutablePointer to optional when it is the actual c-api call site.
+extension UnsafeMutablePointer where Pointee == CTensorHandle? {
+  @usableFromInline
+  init(_ other: UnsafeMutablePointer<CTensorHandle>) {
```
Comment (rxwei) [inline stdlib/public/TensorFlow/Utilities.swift]: Hmm why is this necessary? Isn’t this just doing a copy?

Comment (pschuh) [reply stdlib/public/TensorFlow/Utilities.swift]: Notice Pointee is Optional, but it is taking from a non-optional Pointee.

Comment (rxwei) [reply stdlib/public/TensorFlow/Utilities.swift]: Oh I see.
